content/news/030/index.md

```diff
@@ -76,6 +76,35 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library Updates
 
+### [godot-rust][gd-github]
+
+godot-rust ([GitHub][gd-github], [Discord][gd-discord], [Twitter][gd-twitter])
```

Comment:
maybe add some image or at lesr the project's logo?

Reply:
Would it be OK to showcase some games, like in https://twitter.com/GodotRust/status/1482754475483926531?
Or should I limit it to the godot-rust logo?

Reply:
i think it should be fine (we're already doing a [similar thing](https://gamedev.rs/news/003/#gfx-rs-v0-4) from time to time anyway)